apps/ui/tailwind.config.js

```diff
@@ -59,10 +59,13 @@ module.exports = {
     },
     spacing: {
       0: '0px',
```

Comment:
Are those values part of strict design system or just values that we managed to extract from specific components?

I think it would be great if we could get design team to set a strong definition of those values that will be source of truth for us (like `Text extra small - 14px`, `Text small - 16px`, `Spacing 2 - 8px`.

This way we don't have to guess what the values should be in case there will be new designs that introduce new value that doesn't fit with our predictions of what we should have.

Reply:
You can find it [here](https://www.figma.com/file/PkrjgrUGn4YfsJpnAel6dk/%23foundations?node-id=16%3A10&mode=dev) under typo. Although this is not up to date and in missing 16px and `micro` is never used in design as far as I can see.

And for spacing I think this is a good middle ground as I am always using the exact middle value of the one before and after. `8px` was not changed by me and changing it would call for a large refactor

Reply:
I've worked with the design system quite a lot when integrating into Tune and I think the values we have now are a good start and I also looked over it again and we seem to be cover a good amount of spacing values